src/meta/model/migration/clippy.toml

```diff
@@ -0,0 +1,4 @@
+disallowed-methods = [
+    { path = "sea_query::table::column::ColumnDef::binary", reason = "Please use `.rw_binary(..)` instead." },
```

Comment:
Can you explain it by some comments?

Reply:
Yes, it's explained in docs:

```rust
    /// Set column type as `longblob` for MySQL, `bytea` for Postgres, and `blob` for Sqlite.
    ///
    /// Should be preferred over [`binary`](ColumnDef::binary) or [`blob`](ColumnDef::blob) for large binary fields,
    /// typically the fields wrapping protobuf or other serialized data. Otherwise, MySQL will return an error
    /// when the length exceeds 65535 bytes.
```